src/main/java/com/yahoo/bullet/querying/FilterOperations.java

```diff
@@ -71,6 +73,7 @@
         COMPARATORS.put(Clause.Operation.LESS_EQUALS, isNotNullAnd(LE));
     }
     static final Comparator<Pattern> REGEX_LIKE = isNotNullAnd(RLIKE);
+    static final Comparator<TypedObject> SIZE_OF = isNotNullAnd(SIZEOF);
```

Comment:
Do we want to think of SIZEOF(null) == 0 as true or false? Right now, it's false.

Reply:
We can. But I saw RLIKE(null, ".*") return false, so I added isNotNullAnd.

Reply:
Ok, let's leave it as false unless @NathanSpeidel has a different opinion.